### PR TITLE
Html to pug

### PR DESCRIPTION
**Pull request by BJordi**
## Summary

- Migrated html code to pug in Login and Signup views.

### CODE REVIEW

File: src/views/Login.vue [hunk 1]
```diff
@@ -1,40 +1,21 @@
...
-        >El password debe contener al menos una mayúscula</span>
-      </div>
-      <button class="primary-button">Login</button>
-    </form>
-    <div class="horizontal-division"/>
-    <router-link to="/sign-up" class="secondary-button">Sign up</router-link>
-  </div>
+<template lang="pug">
+  .container
+    img.logo(src='../assets/wolox-logo.png')
```
Comment (FrankIglesias) [inline src/views/Login.vue]: add alt

File: src/views/Login.vue [hunk 2]
```diff
@@ -1,40 +1,21 @@
...
+    form(@submit.prevent='onSubmit')
+      .input-container
+        label.label(for='email') Email
+        input#email(v-model='email', :class="['input', ($v.email.$invalid && submitted) || !$v.email.email ? 'error-input' : '']")
+        span.error-label(v-if='!$v.email.email') El email no tiene un formato válido
+        span.error-label(v-if='!$v.email.required && submitted') Campo requerido
+      .input-container
+        label.label(for='password') Password
+        input#password(type='password', v-model='$v.password.$model', :class="['input', $v.password.$error && ($v.password.required || submitted ) ? 'error-input': '']")
```
Comment (FrankIglesias) [inline src/views/Login.vue]: indent props

File: src/views/Login.vue [hunk 3]
```diff
@@ -1,40 +1,21 @@
...
-    <div class="horizontal-division"/>
-    <router-link to="/sign-up" class="secondary-button">Sign up</router-link>
-  </div>
+<template lang="pug">
+  .container
+    img.logo(src='../assets/wolox-logo.png')
+    form(@submit.prevent='onSubmit')
+      .input-container
+        label.label(for='email') Email
+        input#email(v-model='email', :class="['input', ($v.email.$invalid && submitted) || !$v.email.email ? 'error-input' : '']")
```
Comment (FrankIglesias) [inline src/views/Login.vue]: same here

File: src/views/Login.vue [hunk 4]
```diff
@@ -1,40 +1,21 @@
...
+        span.error-label(v-if='!$v.email.email') El email no tiene un formato válido
+        span.error-label(v-if='!$v.email.required && submitted') Campo requerido
+      .input-container
+        label.label(for='password') Password
+        input#password(type='password', v-model='$v.password.$model', :class="['input', $v.password.$error && ($v.password.required || submitted ) ? 'error-input': '']")
+        span.error-label(v-if='!$v.password.required && submitted') Campo requerido
+        span.error-label(v-if='$v.password.$dirty && $v.password.required && !$v.password.hasNumber') El password debe contener al menos un número
+        span.error-label(v-if='$v.password.$dirty && $v.password.required && !$v.password.hasUppercase') El password debe contener al menos una mayúscula
+      button.primary-button Login
+    .horizontal-division
```
Comment (FrankIglesias) [inline src/views/Login.vue]: use a border instead

File: src/views/SignUp.vue [hunk 5]
```diff
@@ -1,49 +1,28 @@
...
+        span.error-label(v-if='!$v.email.required && submitted') Campo requerido
+      .input-container
+        label.label(for='password') Password
+        input#password(type='password', v-model='$v.password.$model', :class="['input', $v.password.$error && ($v.password.required || submitted ) ? 'error-input': '']")
+        span.error-label(v-if='!$v.password.required && submitted') Campo requerido
+        span.error-label(v-if='$v.password.$dirty && $v.password.required && !$v.password.hasNumber') El password debe contener al menos un número
+        span.error-label(v-if='$v.password.$dirty && $v.password.required && !$v.password.hasUppercase') El password debe contener al menos una mayúscula
+      span.error-label(v-if='formError') {{formError}}
+      button.primary-button Sign up
+    .horizontal-division
```
Comment (FrankIglesias) [inline src/views/SignUp.vue]: same to this whole file

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Comment by FrankIglesias** [comment]
text inputs usually do not have a for prop... so ids can be deleted

File: src/scss/commons/components.scss [hunk 6]
```diff
@@ -56,3 +60,8 @@
 .error-input {
   border: 1px solid $red;
 }
+
+.logo {
+  margin: 25px 0;
+  object-fit: contain;
+}
```
Comment (mcallegari10) [inline src/scss/commons/components.scss]: Add an EOL

File: src/views/Login.vue [hunk 7]
```diff
@@ -1,40 +1,25 @@
...
-    </form>
-    <div class="horizontal-division"/>
-    <router-link to="/sign-up" class="secondary-button">Sign up</router-link>
-  </div>
+<template lang="pug">
+  .container
+    img.logo(src='../assets/wolox-logo.png', alt='wolox-books-logo')
+    form.bottom-separator(@submit.prevent='onSubmit')
+      .input-container
+        label.label(for='email') Email
```
Comment (mcallegari10) [inline src/views/Login.vue]: For plain text, we use pipe notation. For example:

```pug
label(for='email')
  | Email
```

File: src/views/Login.vue [hunk 8]
```diff
@@ -1,40 +1,25 @@
...
-        >El password debe contener al menos una mayúscula</span>
-      </div>
-      <button class="primary-button">Login</button>
-    </form>
-    <div class="horizontal-division"/>
-    <router-link to="/sign-up" class="secondary-button">Sign up</router-link>
-  </div>
+<template lang="pug">
+  .container
+    img.logo(src='../assets/wolox-logo.png', alt='wolox-books-logo')
```
Comment (mcallegari10) [inline src/views/Login.vue]: Instead of doing the attribute separation with commas, just leave a space :smile:

File: src/views/Login.vue [hunk 9]
```diff
@@ -1,40 +1,25 @@
...
+    form.bottom-separator(@submit.prevent='onSubmit')
+      .input-container
+        label.label(for='email') Email
+        input(v-model='email',
```
Comment (mcallegari10) [inline src/views/Login.vue]: No need for the comma

File: src/views/Login.vue [hunk 10]
```diff
@@ -1,40 +1,25 @@
...
+          :class="{'input': true, 'error-input': ($v.email.$invalid && submitted) || !$v.email.email}")
```
Comment (mcallegari10) [inline src/views/Login.vue]: The correct way to indent should be:

```pug
input(
  v-model='...'
  class='input'
  :class='{ "error-input": ($v.email.$invalid && submitted) || !$v.email.email }'
)
```

Check that you don't need to add to the class bindings the ones that you always want to be apply

File: src/views/Login.vue [hunk 11]
```diff
@@ -1,40 +1,25 @@
...
+        span.error-label(v-if='!$v.email.email') El email no tiene un formato válido
+        span.error-label(v-if='!$v.email.required && submitted') Campo requerido
```
Comment (mcallegari10) [inline src/views/Login.vue]: Use pipe notation

File: src/views/Login.vue [hunk 12]
```diff
@@ -1,40 +1,25 @@
...
+      .input-container
+        label.label(for='password') Password
```
Comment (mcallegari10) [inline src/views/Login.vue]: Pipe notation

File: src/views/Login.vue [hunk 13]
```diff
@@ -1,40 +1,25 @@
...
+        input(type='password',
+          v-model='$v.password.$model',
```
Comment (mcallegari10) [inline src/views/Login.vue]: Check the indentation and commas

File: src/views/Login.vue (hunk 9 above)
Comment (mcallegari10) [reply src/views/Login.vue]: Also, remember to put the input type :smile:

File: src/views/Login.vue [hunk 14]
```diff
@@ -1,40 +1,25 @@
...
+          :class="{'input': true, 'error-input': $v.password.$error && ($v.password.required || submitted )}")
+        span.error-label(v-if='!$v.password.required && submitted') Campo requerido
```
Comment (mcallegari10) [inline src/views/Login.vue]: Pipe notation

File: src/views/Login.vue [hunk 15]
```diff
@@ -1,40 +1,25 @@
...
+        span.error-label(v-if='$v.password.$dirty && $v.password.required && !$v.password.hasNumber')
+          | El password debe contener al menos un número
+        span.error-label(v-if='$v.password.$dirty && $v.password.required && !$v.password.hasUppercase')
+          | El password debe contener al menos una mayúscula
+      button.primary-button Login
```
Comment (mcallegari10) [inline src/views/Login.vue]: Pipe notation

File: src/views/Login.vue [hunk 16]
```diff
@@ -1,40 +1,25 @@
...
+    router-link.secondary-button(to='/sign-up') Sign up
```
Comment (mcallegari10) [inline src/views/Login.vue]: Pipe notation

File: src/views/SignUp.vue [hunk 17]
```diff
@@ -1,49 +1,32 @@
...
-      </div>
-      <span class="error-label" v-if="formError">{{formError}}</span>
-      <button class="primary-button">Sign up</button>
-    </form>
-    <div class="horizontal-division"/>
-    <router-link to="/login" class="secondary-button">Login</router-link>
-  </div>
+<template lang="pug">
+  .container
+    img.logo(src='../assets/wolox-logo.png', alt='wolox-books-logo')
```
Comment (mcallegari10) [inline src/views/SignUp.vue]: Check the comments of the previous file in this one too :smile:

**Review by mcallegari10 — CHANGES_REQUESTED** [review]

File: src/views/Login.vue (hunk 8 above)
Comment (BJordi) [reply src/views/Login.vue]: Great! :wink::+1:

File: src/views/Login.vue (hunk 10 above)
Comment (BJordi) [reply src/views/Login.vue]: Ooooh, that's cool! Thank you!

Comment (BJordi) [reply src/views/Login.vue]: Btw, should I use double quotes in `error-input` key? :thinking:

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**